tests/src/test/scala/mhtml/HtmlTests.scala

```diff
@@ -186,6 +186,16 @@ class HtmlTests extends FunSuite {
     assert(div.innerHTML == "<div>&amp;&lt;©λ</div>")
   }
 
+  test("EntityRef") {
+    def entityRefs = Group(Seq(
+      EntityRef("amp"), EntityRef("lt"), EntityRef("copy"), EntityRef("lambda")
+    ))
+    val entityDiv = <div>{entityRefs}</div>
+    val div = dom.document.createElement("div")
+    mount(div, entityDiv)
+    assert(div.innerHTML == "<div>&amp;&lt;©λ</div>")
```

Comment:
Could you make that a one-liner in `tests/src/test/scala/mhtml/RenderTests.scala`?